osv/models.py

```diff
@@ -332,18 +332,25 @@ def _tokenize(self, value):
     value_lower = value.lower()
     return re.split(r'\W+', value_lower) + [value_lower]
 
+  def _normalize(self, name):
+    """Normalize package name per https://peps.python.org/pep-0503/#normalized-names"""
+
+    return re.sub(r"[-_.]+", "-", name).lower()
```

Comment:
nit: use single quotes rather than double quotes.

Reply:
Addressed in 47438b76ae7c3b24ee97d85478bc01325d888bb6